source/administration/command-line-tools.rst

```diff
@@ -35,18 +35,37 @@ From the directory where the Mattermost platform is installed, a
     :backlinks: top
     :local:
 
-Mattermost 3.6 and later
-^^^^^^^^^^^^^^^^^^^^^^^^
+Using the CLI
+^^^^^^^^^^^^^
 
-The new CLI tool is supported in Mattermost 3.6 and later. To see available commands in the old CLI tool, see `Mattermost 3.5 and earlier`_.
+To run the CLI commands, you must be in the directory that contains the Mattermost executable. On a default install of Mattermost, the directory is ``/opt/mattermost/bin``. The name of the executable is ``platform``.
```

Comment:
You can also be in the directory above it (`/opt/mattermost`) and call it as `bin/platform`, but I'm 0/5 on if we actually mention that